eyelid-match-ops/src/encrypted/test/matching.rs

```diff
@@ -0,0 +1,119 @@
+//! Encrypted iris matching tests.
+
+use crate::encoded::{PolyCode, PolyQuery};
+use crate::encrypted::{EncryptedPolyCode, EncryptedPolyQuery};
+use crate::iris::conf::IrisConf;
+use crate::plaintext::test::matching::{different, matching};
+use crate::primitives::poly::Poly;
+use crate::primitives::yashe::Yashe;
+use crate::{EncodeConf, FullBits, FullRes, PolyConf, YasheConf};
+use colored::Colorize;
+
+fn convert_negative_coefficients<C: EncodeConf<PlainConf = FullRes>>(
+    polys: &mut [Poly<C::PlainConf>],
+) {
+    for poly in polys {
+        Poly::coeffs_modify_non_zero(poly, |coeff: &mut <C::PlainConf as PolyConf>::Coeff| {
+            let mut coeff_res = C::PlainConf::coeff_as_big_int(*coeff);
+            if coeff_res > <C::PlainConf as YasheConf>::modulus_minus_one_div_two_as_big_int() {
+                coeff_res += C::PlainConf::T;
+                *coeff = C::PlainConf::big_int_as_coeff(coeff_res);
+            }
+        });
+    }
+}
+
+#[test]
+fn test_matching_homomorphic_codes() {
+    matching_codes::<FullBits>();
```

Comment:
Can we run both tests in this file on the full and mid resolutions, to check that we haven't hard-coded anything about the full resolution?

(It's ok if they don't work for the tiny resolution, trying two different configs is enough for test coverage.)

Reply:
it is not working for Middle resolution. But given the change in priorities I am in favor of creating an issue to solve it in the future.

Reply:
Sure, let's do that.

Reply:
This is now ticket #112